src/Entity/EmailTemplate/EmailTemplate.php

```diff
@@ -0,0 +1,154 @@
+ *             "normalization_context": {
+ *                 "groups": {"email_template_list_read"}
+ *             },
+ *         },
+ *         "post": {
+ *             "path": "/v3/email_templates",
+ *         },
+ *     },
+ *     itemOperations={
+ *         "get": {
+ *             "path": "/v3/email_templates/{id}",
+ *             "requirements": {"id": "%pattern_uuid%"},
+ *             "access_control": "is_granted('ROLE_MESSAGE_REDACTOR') and object.getAuthor() == user",
+ *             "swagger_context": {
+ *                 "parameters": {
+ *                     {
+ *                         "name": "id",
+ *                         "in": "path",
+ *                         "type": "uuid",
+ *                         "description": "The UUID of the Email Template resource.",
+ *                         "example": "de7982c4-3729-4f9d-9587-376df25354c3",
+ *                     },
+ *                 },
+ *             },
+ *         },
+ *         "put": {
+ *             "path": "/v3/email_templates/{id}",
+ *             "requirements": {"id": "%pattern_uuid%"},
+ *             "access_control": "is_granted('ROLE_MESSAGE_REDACTOR') and object.getAuthor() == user",
+ *         },
+ *         "delete": {
+ *             "path": "/v3/email_templates/{id}",
+ *             "requirements": {"id": "%pattern_uuid%"},
+ *             "access_control": "is_granted('ROLE_MESSAGE_REDACTOR') and object.getAuthor() == user",
+ *             "swagger_context": {
+ *                 "parameters": {
+ *                     {
+ *                         "name": "id",
+ *                         "in": "path",
+ *                         "type": "uuid",
+ *                         "description": "The UUID of the Email Template resource.",
+ *                         "example": "de7982c4-3729-4f9d-9587-376df25354c3",
+ *                     },
+ *                 },
+ *             },
+ *         },
+ *     },
+ * )
+ *
+ * @ORM\Entity
+ * @ORM\Table(
+ *     name="email_templates",
+ *     uniqueConstraints={
+ *         @ORM\UniqueConstraint(name="email_template_uuid_unique", columns="uuid")
+ *     }
+ * )
+ * @ORM\AssociationOverrides({
+ *     @ORM\AssociationOverride(name="author",
+ *         joinColumns={
+ *             @ORM\JoinColumn(onDelete="SET NULL")
+ *         }
+ *     )
+ * })
+ */
+class EmailTemplate implements AuthorInterface
+{
+    use EntityIdentityTrait;
+    use EntityTimestampableTrait;
+    use AuthoredTrait;
+
+    /**
+     * @var string|null
+     *
+     * @ORM\Column
+     *
+     * @SymfonySerializer\Groups({"email_template_read", "email_template_write", "email_template_list_read"})
+     *
+     * @Assert\NotBlank
+     * @Assert\Length(max="255")
+     */
+    private $label;
+
+    /**
+     * @var string|null
+     *
+     * @ORM\Column(type="text")
+     *
+     * @Assert\NotBlank
+     *
+     * @SymfonySerializer\Groups({"email_template_read", "email_template_write", "email_template_list_read"})
```

Comment:
could you remove `email_template_list_read` please, we don't really need to have the big content on list page